web/themes/custom/tube_theme/css/style.css

```diff
@@ -7703,6 +7703,8 @@ nav.tabs {
     background-color: #FB8920; }
```

Comment:
Please aim to commit the CSS and SCSS files separately. In this case, I would have merged in only the scss, because the css has merge conflicts with other teammate's PR's. I'll resolve the merge conflict here, but it's better to make it easy to avoid :)

Reply:
That comment was for this commit: Added css and twig to make shapes square or round